### PR TITLE
Update AXL manuscript

### PR DESCRIPTION
**Pull request by mcreixell**
Updated all figures and added their corresponding captions. Two outstanding items:

1. The idea of figure 2 is to provide an overview of the phosphoproteomic data using currently available tools. That's why I added an enrichment analysis with KSEA of PC9 WT vs KO. I was thinking of adding a volcano plot with p-values comparing both cell lines as well. It still looks incomplete to me, any suggestions?

2.  In figure 4, for some reason when I overlay both blots `YAP_WTvsKO_blot.svg` overrides `YAP_DR_blot.svg` and part of the former occupies the subplot. However, when I comment out one of the two to display only one at a time, both work as they're supposed to. Do you have any idea of why this is happening?

### CODE REVIEW

File: msresist/tests/test_CoClustering.py [hunk 1]
```diff
@@ -50,4 +50,4 @@ def test_pickle(distm):
     _, scores, _, _ = EM_clustering(data, info, 5, gmmIn=unpickled.gmm_)
 
     assert np.all(np.isfinite(unpickled.scores_))
-    np.testing.assert_allclose(MSC.scores_, scores, rtol=1e-2, atol=0.1)
+    np.testing.assert_allclose(MSC.scores_, scores, rtol=0.1, atol=0.2)
```
Comment (mcreixell) [inline msresist/tests/test_CoClustering.py:53]: @aarmey I'm not sure if this is too high? This test was always failing and it still does some times with these new tolerance settings.

Comment (aarmey) [reply msresist/tests/test_CoClustering.py:53]: This should be fine. It looks like it fails due to a very small number of peptide assignments changing.

**Comment by aarmey** [comment]
Regarding #1, a whole figure is a lot of space to dedicate to something we don't see as a major point of the paper. I'd rather that we quickly get to analysis of meaningful results.

I don't see figure 4 in the build? Perhaps you could post images here of when it is working with one placement, and broken with both?

File: makefile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-flist = 1 2 3 S3 M2 M3 M4 M5 MS2 MS3 MS4 MS5 MS6 MS7
+flist = 1 2 3 S1 S2 S3 S4 S5 S3 M2 M3 M4 M5 MS2 MS3 MS4 MS5 MS6 MS7
```
Comment (aarmey) [inline makefile:1]: Add 4 here.

File: msresist/figures/figure1.py [hunk 3]
```diff
@@ -30,7 +30,7 @@
 def makeFigure():
     """Get a list of the axis objects and create a figure"""
     # Get list of axis objects
```
Comment (aarmey) [inline msresist/figures/figure1.py:32]: The PCA results are just shoved in at the end, when I think they are really useful to synthesize all the phenotypic measurements. I'd suggest: (1) it's just the PCA results in Fig. 1, with select examples after, and all the phenotypic measurements are in the supplement, or (2) there's a figure on measuring various phenotypes, and then the PCA analysis is a separate Fig. 2.

There probably needs to be a cartoon at the beginning introducing the AXL KO and reintroduction, and small schematics describing each of the phenotypic measurements.

File: msresist/figures/figure2.py [hunk 4]
```diff
@@ -16,7 +16,7 @@
 def makeFigure():
     """Get a list of the axis objects and create a figure"""
     # Get list of axis objects
```
Comment (aarmey) [inline msresist/figures/figure2.py:18]: Seems like the KSEA is obviously not very informative?

File: msresist/figures/figureS1.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
+"""
+This creates Supplemental Figure 1: Cell Viability time course
```
Comment (aarmey) [inline msresist/figures/figureS1.py:2]: Seems to repeat quite a few plots in Fig. 1?

File: msresist/figures/figureS2.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
+"""
+This creates Supplemental Figure 2: Cell M time course
+"""
+
```
Comment (aarmey) [inline msresist/figures/figureS2.py:4]: Same as with S1. Also, many journals limit the number of supplementary figures now to either supplementary figures must correspond to one of the main figures, or there can be no greater number than main figures. Should each phenotype have its own supplementary figure? Are you going to reference these plots in the text?

File: msresist/figures/figureS3.py [hunk 7]
```diff
@@ -1,77 +1,30 @@
 """
-This creates Supplemental Figure 3: Prediction performance across different plsr models using different clustering strategies
+This creates Supplemental Figure 3: Cell Migration time course
```
Comment (aarmey) [inline msresist/figures/figureS3.py:2]: Same here.

File: msresist/figures/figureS4.py [hunk 8]
```diff
@@ -0,0 +1,44 @@
+"""
+This creates Supplemental Figure 4: Island effect across radiis and time points
```
Comment (aarmey) [inline msresist/figures/figureS4.py:2]: Same here.

File: msresist/figures/figureS5.py [hunk 9]
```diff
@@ -0,0 +1,76 @@
+"""
+This creates Supplemental Figure 3: Prediction performance across different plsr models using different clustering strategies
```
Comment (aarmey) [inline msresist/figures/figureS5.py:2]: I think this information should be in the main figure and we don't need a supplementary one.

- The performance versus component number can go—readers don't need to see this information if you're properly picking the right number of components.
- All the different methods can be presented in one plot using a stripplot like this:

**Review by aarmey — CHANGES_REQUESTED** [review]

**Comment by mcreixell** [comment]
figure4
figure4-1
figure4-2

**Comment by mcreixell** [comment]
Working on the rest of the edits now

**Comment by aarmey** [comment]
Not sure what's going on with those placements. We can work on merging this ignoring that, then I'll look into it. There's probably something weird about how the placement code is handling scaling that will take a bit to diagnose.

**Comment by mcreixell** [comment]
Same thing happens with figure 1. I think this problem with the placements occurs when trying to overlay two .svg files contiguously. I couldn't come up with a diagram that made sense to describe the phenotypes but I think that cell viability / death are self-explanatory and that these images perfectly show what migration and island effect are.   

figure1
figure1-1

**Comment by mcreixell** [comment]
@aarmey I will update the figure captions once you've reviewed these last round of edits. I split the supplementals of the phenotypes in two figures because each time lapse panel consists of 10 plots x 4 phenotypes = 40 plots + 4 bar plots equals 44 plots. I split them in 22 (viability&death) and 22 (migration&island).

**Review by aarmey — APPROVED** [review]

**Pull request merged**